src/shimoku_api_python/api/plot_api.py

```diff
@@ -7672,6 +7672,157 @@ def line_with_confidence_area(self,
+                    if k in updated_mapping and isinstance(updated_mapping[k], dict) and isinstance(v, dict):
+                        updated_mapping[k] = deep_update(updated_mapping[k], v)
+                    else:
+                        updated_mapping[k] = v
+            return updated_mapping
+
+        # Default echarts options for tornado
+        pos_dataops_org = {
+            'label': {
+                'position': 'right',
+            },
+            'itemStyle': {
+                'color': 'var(--chart-C2)',
+                'borderRadius': [0, 9, 9, 0],
+            },
+            'emphasis': {
+                'itemStyle': {
+                    'color': 'var(--chart-C2)',
+                    'borderColor': 'var(--chart-C2)',
+                }
+            },
+        }
+
+        neg_dataops_org = {
+            'label': {
+                'position': 'left',
+            },
+            'itemStyle': {
+                'color': 'var(--complementary-red-light)',
+                'borderRadius': [9, 0, 0, 9],
+            },
+            'emphasis': {
+                'itemStyle': {
+                    'color': 'var(--complementary-red-light)',
+                    'borderColor': 'var(--complementary-red-light)',
+                }
+            },
+        }
+
+        options_org = {
+            'toolbox': {
+                'right': 10,
+                'feature': {
+                    'saveAsImage': {},
+                    'dataView': {},
+                }
+            },
+            'tooltip': {
+                'trigger': 'axis',
+                'axisPointer': {
+                    'type': 'shadow'
+                }
+            },
+            'grid': {
+                'top': 80,
+                'bottom': 30
+            },
+            'xAxis': {
+                'type': 'value',
+                'axisLine': {'show': False},
+                'position': 'top',
+                'axisLabel': {'show': False},
+                'splitLine': {
+                    'show': False,
+                }
+            },
+            'yAxis': {
+                'type': 'category',
+                'axisLine': {'show': True},
+                'axisLabel': {'show': False},
+                'axisTick': {'show': False},
+                'splitLine': {'show': False},
+            },
+        };
+
+        seriesopt_org = {
+            'name': 'Revenue',
+            'type': 'bar',
+            'stack': 'Total',
+            'smooth': True,
+            'label': {
+                'show': True,
+                'width': 270,
+                'overflow': 'truncate',
+                'formatter': '{b}\n{c}',
+            },
+        }
+
+        options = deep_update(options_org, options_mod)
+        pos_dataops = deep_update(pos_dataops_org, pos_dataops_mod)
+        neg_dataops = deep_update(neg_dataops_org, neg_dataops_mod)
+        series = deep_update(seriesopt_org, seriesopt_mod)
```

Comment:
Same comment as in the multiple user defined modification parameters, the chart options can be defined in one dictionary, see the stacked barchart for example on how to handle multiple series with dynamic data, it takes the series by order, in this case we only have two series